### PR TITLE
Tidy up validate-modules ignores for modules: cloud/oneandone

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Tidy up on sanity tests' ignore files, removing lines with validation-modules for these modules.

The focus was to pass the tests so no code was changed other than adjusting argument_spec for missing types or inconsistencies with the docs. The priority has been not to change any functionality, but to ensure that code and docs are synchronized.

Some lines were added back to the files, but a total of 57 lines were removed (among 2.9, 2.10 and 2.11).

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
plugins/modules/cloud/oneandone/oneandone_server.py
plugins/modules/cloud/oneandone/oneandone_public_ip.py
plugins/modules/cloud/oneandone/oneandone_private_network.py
plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py
plugins/modules/cloud/oneandone/oneandone_load_balancer.py
plugins/modules/cloud/oneandone/oneandone_firewall_policy.py

### CODE REVIEW

**Comment by russoz** [comment]
recheck

File: plugins/modules/cloud/oneandone/oneandone_server.py [hunk 1]
```diff
@@ -28,80 +28,98 @@
...
         It is required only for 'present' state.
+    type: str
   fixed_instance_size:
     description:
       - The instance size name or ID of the server.
         It is required only for 'present' state, and it is mutually exclusive with
         vcore, cores_per_processor, ram, and hdds parameters.
-    required: true
-    choices: [ "S", "M", "L", "XL", "XXL", "3XL", "4XL", "5XL" ]
+      - 'The available choices are: I(S), I(M), I(L), I(XL), I(XXL), I(3XL), I(4XL), I(5XL)'
```
Comment (felixfontein) [inline plugins/modules/cloud/oneandone/oneandone_server.py:68]:
```suggestion
      - 'The available choices are: C(S), C(M), C(L), C(XL), C(XXL), C(3XL), C(4XL), C(5XL)'
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks for fixing this!

**Comment by russoz** [comment]
@felixfontein thank you!

**Pull request merged**